### PR TITLE
Remove enterprise from upgrade notes

### PR DESCRIPTION
**Pull request by astuyve**
## What did you implement:

I changed the wording of the upgrade message from Enterprise to Dashboard, and added `the`

## How did you implement it:

By deleting and adding text

## How can we verify it:

You can pull this branch with an outdated version of the enterprise-plugin and see this message

**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO

### CODE REVIEW

File: lib/classes/PluginManager.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ class PluginManager {
       const updates = updateNotifier({ pkg: sfePkgJson, interval: 1 });
       if (updates.update) {
         this.serverless.cli.log(
-          'An updated version of Serverless Enterprise is available. ' +
+          'An updated version of the Serverless Dashboard is available. ' +
```
Comment (pmuens) [inline lib/classes/PluginManager.js]: While this might be nitpicking I think that this message sounds a little bit strange.

Our Dashboard is a SaaS offering, so why do I need to update the Serverless Framework when the hosted Dashboard was updated? I'd rephrase it to state that the underlying package which interacts with the Dashboard needs to be updated.

I know that this might be too much, but when reading it it just doesn't feel right.

**Comment by astuyve** [comment]
Thanks @pmuens! I didn't get time to reply yesterday, but essentially we still wanted some way to identify if the plugin was out of date but refer to it cohesively with the rest of our marketing and branding.
